projects/v2/src/app/components/organ-table-selector/organ-table-selector.component.html

```diff
@@ -0,0 +1,61 @@
+
+<h2 mat-dialog-title>Select one or more ASCT+B Tables</h2>
```

Comment:
This should be a table in a modal, not a dropdown. A table will allow you to show more organs at once.

Reply:
Also, they can only choose one version (they may not choose multiple versions of the same organ)